server/src/main/java/org/apache/cloudstack/vm/VmImportManagerImpl.java

```diff
@@ -428,13 +428,13 @@ private ServiceOfferingVO getUnmanagedInstanceServiceOffering(final UnmanagedIns
             serviceOffering = serviceOfferingDao.getComputeOffering(serviceOffering, parameters);
         } else {
             if (!cpu.equals(serviceOffering.getCpu()) && !instance.getPowerState().equals(UnmanagedInstanceTO.PowerState.PowerOff)) {
-                throw new ServerApiException(ApiErrorCode.INTERNAL_ERROR, String.format("Service offering (%s) %d CPU cores does not matches VM CPU cores %d  and VM is not in powered off state (Power state: %s)", serviceOffering.getUuid(), serviceOffering.getCpu(), cpu, instance.getPowerState()));
+                throw new ServerApiException(ApiErrorCode.INTERNAL_ERROR, String.format("Service offering (%s) %d CPU cores does not match VM CPU cores %d  and VM is not in powered off state (Power state: %s)", serviceOffering.getUuid(), serviceOffering.getCpu(), cpu, instance.getPowerState()));
```

Comment:
Additionally, considering the fact that **CPU cores** is plural (of a _CPU core_) it should be "CPU core**s do** not **match**"

Reply:
Hi Gabriel,  Thank you very much for the review. I made the 2 changes as per the review. I will submit the pull request shortly.